src/site/content/en/blog/sign-up-form-best-practices/index.md

```diff
@@ -288,7 +288,7 @@ You should also provide your users with access to their account login history, s
 and when a login happened.
 
 <figure class="w-figure">
-  <img src="images/gmail-activity.jpg" alt="Gmail account activity page">
+  ![Gmail account activity page](images/gmail-activity.jpg)
```

Comment:
All of the images in this file were missed

Reply:
This was missed because the image isn't in the same folder as the markdown, I'll get this in another batch of uploads.